TutorLizard.BusinessLogic/Models/User.cs

```diff
@@ -11,5 +13,25 @@ public class User
     public int Id { get; set; }
     public string Name { get; set; }
     public UserType UserType { get; set; }
+
+    public User(int id, string name, UserType type)
+    {
+        this.Id = id;
+        this.Name = name;
+        this.UserType = type;
+    }
+    public static int GetID()
```

Comment:
I don't think this method belongs in this class. We'll be loading list of users at startup in DataAccess anyway, so let's just just add a private method there that gets this from this list using LINQ.
I don't think that anything outside of DataAccess needs this information.

Reply:
Good point, only DataAccess should has an access to files. Even if id is part of this model, in that kind of architecure we are supposed to avoid these inputs